core/domain/topic_services_test.py

```diff
@@ -635,7 +1044,7 @@ def test_invalid_publish_and_unpublish_story(self) -> None:
             topic_services.unpublish_story(
                 self.TOPIC_ID, 'story_10', self.user_id_admin)
 
-        # Throw error if a story node doesn't have an exploration.
+        # Throw error if exploration isn't published.
```

Comment:
I am a bit confused by this comment. What exactly is causing an error to be thrown here?

Reply:
Actually, I think the original comment answers your question much better. My bad. I'll revert this change.